### ISSUE TITLE
cannot delete room

### ISSUE THREAD

**Issue by majestrate** [oxen-io/session-pysogs#153]
version: `PySOGS 0.3.5`

config additions, otherwise stock settings.

```ini
[users]                                                                                                                                                                                                            
require_blind_keys = yes   
```

traceback given while trying to delete a room

```
The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3/dist-packages/sogs/__main__.py", line 341, in <module>
    room.delete()
  File "/usr/lib/python3/dist-packages/sogs/model/room.py", line 186, in delete
    result = query("DELETE FROM rooms WHERE token = :t", t=self.token)
  File "/usr/lib/python3/dist-packages/sogs/db.py", line 54, in query
    return dbconn.execute(q, **params)
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1011, in execute
    return meth(self, multiparams, params)
  File "/usr/lib/python3/dist-packages/sqlalchemy/sql/elements.py", line 298, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1124, in _execute_clauseelement
    ret = self._execute_context(
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1316, in _execute_context
    self._handle_dbapi_exception(
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1510, in _handle_dbapi_exception
    util.raise_(
  File "/usr/lib/python3/dist-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/default.py", line 609, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.IntegrityError: (sqlite3.IntegrityError) FOREIGN KEY constraint failed
[SQL: DELETE FROM rooms WHERE token = ?]
[parameters: ('roomgoeshere',)]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
```

**Comment by jagerman** [comment]
I'm not able to reproduce this, so going to tag v0.3.6 without a fix for it.

**Comment by DayInTheLifeOfPhil** [comment]
Can reproduce this in same version if room on server has capital letter in name

**Comment by ranadumbknown** [comment]
I have the same identical problem deleteing a room. my room name has a dash in it.  - is there a way to manually delete or at least break the room to make it where no one can join?

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1245, in _execute_context
    self.dialect.do_execute(
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/default.py", line 581, in do_execute
    cursor.execute(statement, parameters)
sqlite3.IntegrityError: FOREIGN KEY constraint failed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3/dist-packages/sogs/__main__.py", line 341, in <module>
    room.delete()
  File "/usr/lib/python3/dist-packages/sogs/model/room.py", line 186, in delete
    result = query("DELETE FROM rooms WHERE token = :t", t=self.token)
  File "/usr/lib/python3/dist-packages/sogs/db.py", line 54, in query
    return dbconn.execute(q, **params)
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 982, in execute
    return meth(self, multiparams, params)
  File "/usr/lib/python3/dist-packages/sqlalchemy/sql/elements.py", line 287, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1095, in _execute_clauseelement
    ret = self._execute_context(
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    self._handle_dbapi_exception(
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1476, in _handle_dbapi_exception
    util.raise_from_cause(sqlalchemy_exception, exc_info)
  File "/usr/lib/python3/dist-packages/sqlalchemy/util/compat.py", line 398, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/usr/lib/python3/dist-packages/sqlalchemy/util/compat.py", line 152, in reraise
    raise value.with_traceback(tb)
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1245, in _execute_context
    self.dialect.do_execute(
  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/default.py", line 581, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.IntegrityError: (sqlite3.IntegrityError) FOREIGN KEY constraint failed
[SQL: DELETE FROM rooms WHERE token = ?]
[parameters: ('nsfw-archive',)]
(Background on this error at: http://sqlalche.me/e/gkpj)

**Comment by Acbn-Nick** [comment]
I am seeing this too, error looks the same as above

**Comment by jagerman** [comment]
Fixed by #166
